src/ui.cpp

```diff
@@ -829,6 +829,11 @@ void uimenu::reset()
         wrefresh(window);
         delwin(window);
         window = NULL;
+
+        // window area is lost when owner of uimenu exits its function for SDL
+        #ifdef TILES
+        g->refresh_all();
```

Comment:
This needs to be removed. It causes the crash issue. The uimenu is used from the character generation screen, and this calls `game::refresh_all`, which redraws the terrain, but the whole map is not yet loaded, so nothing can be drawn. Specifically, drawing requires the submaps to be loaded at is looks up the terrain and so on there, but the map is not loaded - `map::load` has not yet been called.